tools/records-downloader-tool/src/main/java/eu/europeana/cloud/EuropeanaRecordsDownloaderTool.java

```diff
@@ -0,0 +1,78 @@
+package eu.europeana.cloud;
+
+import eu.europeana.cloud.downloader.RecordDownloader;
+import eu.europeana.cloud.exception.RepresentationNotFoundException;
+import eu.europeana.cloud.mcs.driver.DataSetServiceClient;
+import eu.europeana.cloud.mcs.driver.FileServiceClient;
+import eu.europeana.cloud.mcs.driver.exception.DriverException;
+import eu.europeana.cloud.service.mcs.exception.MCSException;
+import eu.europeana.cloud.service.mcs.exception.RepresentationNotExistsException;
+import eu.europeana.cloud.util.CommandLineHelper;
+import eu.europeana.cloud.util.FolderCompressor;
+import org.apache.commons.cli.*;
+import org.apache.tika.mime.MimeTypeException;
+import org.zeroturnaround.zip.ZipException;
+
+import java.io.IOException;
+
+
+/**
+ * Created by Tarek on 9/1/2016.
+ */
+public class EuropeanaRecordsDownloaderTool {
+
+    private final static String MCS_URL = "mcsUrl";
+    private final static String USER = "username";
+    private final static String PASSWORD = "password";
+    private final static String PROVIDER_ID = "provider";
+    private final static String DATASET_NAME = "dataset";
+    private final static String REPRESENTATION_NAME = "representation";
+
+    public static void main(String[] args) {
+        Options options = getRequiredOptions();
+        CommandLineParser parser = new DefaultParser();
+        try {
+            CommandLine cmd = parser.parse(options, args);
+            String folderPath = executeDownloader(cmd);
+            System.out.println("The download completed successfully and the zip folder is located : " + FolderCompressor.compress(folderPath));
```

Comment:
Compressed file is put to temp with random name.

The download completed successfully and the zip folder is located : /tmp/10d3a94a-34d7-4e8d-aafb-b6df7e601eca5107900615026729283.zip